iron-image.html

```diff
@@ -396,8 +389,14 @@
       },
 
       _resolveSrc: function(testSrc) {
-        var baseURI = /** @type {string} */(this.ownerDocument.baseURI);
-        return (new URL(Polymer.ResolveUrl.resolveUrl(testSrc, baseURI), baseURI)).href;
+        var resolved = Polymer.ResolveUrl.resolveUrl(testSrc, this.$.baseURIAnchor.href);
```

Comment:
So the idea here is that we want the element to respect whatever base URI applies to the `iron-image` rather than always being relative to the import in which it's used, for example. Regular `image` does this so it seemed reasonable to follow suit.